akka-cluster/src/multi-jvm/scala/akka/cluster/StreamRefSpec.scala

```diff
@@ -43,7 +44,7 @@ object StreamRefSpec extends MultiNodeConfig {
 
   testTransport(on = true)
 
-  case class RequestLogs(streamId: Int)
+  case class RequestLogs(streamId: Int) extends CborSerializable
   case class LogsOffer(streamId: Int, sourceRef: SourceRef[String])
```

Comment:
this one isn't serialised?

Reply:
oversight, and actually turns out serialization of stream refs are not well supported at all: #27304